Improvement for Parser.addini to facilite parsing of 'int' and 'float' values (#13193)
pytest-dev/pytest: changelog/11381.improvement.rst

> The ``type`` parameter of the ``parser.addini`` method now accepts the **int** and **float** literals, facilitating the parsing of configuration values in the **ini** file.

Reviewer: ```suggestion
The ``type`` parameter of the ``parser.addini`` method now accepts `"int"` and ``"float"`` parameters, facilitating the parsing of configuration values in the configuration file.
```

Example:

> .. code-block:: pytest

Reviewer: ```suggestion
.. code-block:: python
```

  def pytest_addoption(parser):
      parser.addini(
          "int_value",
          type="int",
          default=2,
          help="my int value"
      )
      parser.addini(
          "float_value",
          type="float",
          default=4.2,
          help="my float value"
      )

> The **pytest.ini** file:

Reviewer: ```suggestion
The `pytest.ini` file:
```

> .. code-block:: pytest

Reviewer: ```suggestion
.. code-block:: ini
```
